src/tradable/cmn/Roles.sol

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+library ElvCmnRoles {
```

Comment:
Is there a benefit in adding the subdirectory `cmn/`?  Would there be a roles file that is not common inside `tradable/`

Reply:
`Redeemable.sol` uses minter_role as well so added as common roles.